### PR TITLE
re-adds grace ascension, fixes returning to FALL ASLEEP

### PR DESCRIPTION
**Pull request by Militaires**
this BOTH re-adds ascension and fixes the issue highlighted 

TLDR:

chrome_2017-03-11_13-17-07
chrome_2017-03-11_13-17-27

687474703a2f2f692e696d6775722e636f6d2f6177704275456d2e676966
chrome_2017-03-11_18-11-30

chrome_2017-03-11_13-17-54

talk about talking big on improve not remove and then turning the other way cause "maintainers like to suck each other's dicks"

#24809

### CODE REVIEW

**Comment by ChangelingRain** [comment]
Honestly this is stupid, using his grace should only be a drawn-out suicide, and it's balanced with the idea that, if you buy it, you're going to fucking die and the only question is "when".

**Comment by Militaires** [comment]
@changelingrain this means it will 100% be on bloodlust and kill yo ass and also never sleep ngga

File: code/game/objects/items/weapons/his_grace.dm [hunk 1]
```diff
@@ -76,7 +76,7 @@
 	if(!bloodthirst)
 		drowse()
 		return
-	if(bloodthirst < HIS_GRACE_CONSUME_OWNER)
+	if(bloodthirst < HIS_GRACE_CONSUME_OWNER && !ascended)
 		adjust_bloodthirst(1 + Floor(LAZYLEN(contents) * 0.5)) //Maybe adjust this?
 	else
 		adjust_bloodthirst(1) //don't cool off rapidly once we're at the point where His Grace consumes all.
```
Comment (ChangelingRain) [inline code/game/objects/items/weapons/his_grace.dm]: I mean no you'll still have increasing bloodlust, you'll just have literally none of the threat unless there's fuckall for people remaining

Comment (Militaires) [inline code/game/objects/items/weapons/his_grace.dm]: ye but by the time you reach 20 bodies the hunger to kill owner speed is almost instant though

**Comment by optimumtact** [comment]
This doesn't need to be adjusted

**Comment by coiax** [comment]
@optimumtact do you understand what happens to an ascended his grace?

**Comment by Militaires** [comment]
pushing commits

**Comment by ChangelingRain** [comment]
So this both doesn't fix #24809 AND it instantly kills you AND will one-hit and consume anyone unarmored who walks by it?

**Comment by Militaires** [comment]
on second thought no cause you can just laser it and it dies

**Comment by Militaires** [comment]
woops forgot that line

**Comment by KorPhaeron** [comment]
Ascension is gone

**Comment by Militaires** [comment]
@KorPhaeron cause you removed not imrpoved

**Comment by KorPhaeron** [comment]
fine joan can close it then

**Comment by Militaires** [comment]
i fucked shit up with other files let me fix

**Comment by ChangelingRain** [comment]
Ascension is a bad mechanic even if a guy with a laser can kill the toolbox because literally nobody else can

**Comment by Militaires** [comment]
@ChangelingRain

i wont even try to explain how easy it is to acquire a laser gun cause you dont play the game.

Ascension is a bad mechanic even if a guy with a **crematorium** can kill the toolbox because literally nobody else can

Ascension is a bad mechanic even if a guy with a **bomb** can kill the toolbox because literally nobody else can

Ascension is a bad mechanic even if a guy with a **singulo** can kill the toolbox because literally nobody else can

your points are pretty flimsy friend

**Comment by Haevacht** [comment]
What fucking world are you on where randomass crew has good acces to lasers.

**Comment by kevinz000** [comment]
I guess round lengths are still too long
Better add an item that gets better as you murderbone. Murderbone enough and it becomes a permanent sentient dualsword!

**Comment by Haevacht** [comment]
And it kills people after you died too kev, and eats corpses!

**Comment by Militaires** [comment]
@haevacht literally any round sci is doing its job there are energy guns out the ass H 

i mean even if lasers are hard to get what joan doesnt see  is that it's not difficult to say "ascended his grace chapel sec please come laser it"

which is ridiculous in and of itself

**Comment by Haevacht** [comment]
You love that thumbs down button.

**Comment by optimumtact** [comment]
In retrospect the ascension aspect was a little strong, but I originally held my tongue on the port because I wanted to see how it shook out.

Since the maintainers removed it I have to assume it was a little over powered.

This item is still unique and interesting without the ascension so I think we can safely remove it.

**Comment by Militaires** [comment]
@haevacht i thought you'd love it considering how many thumbs go in your ass a day

@optimumtact only joan removed it because they think that his grace ascending is overpowered and makes it impossible to deal with when you can laser it and it gets QDEL'd

man fuck this gay earth of maintainers closing the pr after just assuming

**Pull request closed without merging**